### PR TITLE
Doc fix: Correct hubble exporter config lines

### PR DESCRIPTION
**Pull request by saintdle**
Fixes Hubble exporter doc where incorrect case is used on some of the config lines to configure the feature. This incorrect case leads to errors in the cilium-agent and the files cannot be wrote. 

Added information on how to disable the static exporter config #30425

### CODE REVIEW

File: Documentation/observability/hubble-exporter.rst [hunk 1]
```diff
@@ -235,8 +235,8 @@ You can change flow log settings without a need for pod to be restarted
       --set hubble.export.dynamic.enabled=true \\
       --set hubble.export.dynamic.config.content[0].name=system \\
       --set hubble.export.dynamic.config.content[0].filePath=/var/run/cilium/hubble/events-system.log \\
-      --set hubble.export.dynamic.config.content[0].IncludeFilters[0].source_pod[0]='kube_system/*' \\
-      --set hubble.export.dynamic.config.content[0].IncludeFilters[1].destination_pod[0]='kube_system/*'
+      --set hubble.export.dynamic.config.content[0].includeFilters[0].source_pod[0]='kube_system/*' \\
+      --set hubble.export.dynamic.config.content[0].includeFilters[1].destination_pod[0]='kube_system/*'
```
Comment (AwesomePatrol) [inline Documentation/observability/hubble-exporter.rst]: We can use this opportunity to also fix asterisk (`*`). Either remove it (as this filter prefix matches ns+pod_name) or escape it (`\*`) so the lower line won't be in italics.

Comment (saintdle) [reply Documentation/observability/hubble-exporter.rst]: done

**Review by AwesomePatrol — APPROVED** [review]

**Comment by nathanjsweet** [comment]
I'm adding someone from hubble for review.

**Review by chancez — APPROVED** [review]

File: Documentation/observability/hubble-exporter.rst [hunk 2]
```diff
@@ -47,7 +47,13 @@ Verify that flow logs are stored in target files:
 
     kubectl -n kube-system exec ds/cilium -- tail -f /var/run/cilium/hubble/events.log
 
-Now you can configure your logging solution to consume logs from your Hubble export file path.
+Once you have configured the Hubble Exporter, you can configure your logging solution to consume logs from your Hubble export file path.
+
+To disable the static configuration, you must remove the ``hubble-export-file-path`` key in the ``cilium-config`` ConfigMap and manually clean up the log files created in the specified location in the container. The below command will restart the Cilium pods, if you edit the ConfigMap manually, you will need to restart the Cilium pods.
```
Comment (learnitall) [inline Documentation/observability/hubble-exporter.rst]: Small punctuation suggestion. I feel like this one sentence could be split into two:

```suggestion
To disable the static configuration, you must remove the ``hubble-export-file-path`` key in the ``cilium-config`` ConfigMap and manually clean up the log files created in the specified location in the container. The below command will restart the Cilium pods. If you edit the ConfigMap manually, you will need to restart the Cilium pods.
```

**Review by learnitall — CHANGES_REQUESTED** [review]
One small punctuation nit. Also, would you mind wrapping your lines? I'm not sure if we have a specific max line length, but it would be great if we could keep each line to a maximum of around 90 to 100 characters.

Otherwise LGTM! Thanks!

File: Documentation/observability/hubble-exporter.rst [hunk 3]
```diff
@@ -47,7 +47,13 @@ Verify that flow logs are stored in target files:
 
     kubectl -n kube-system exec ds/cilium -- tail -f /var/run/cilium/hubble/events.log
 
-Now you can configure your logging solution to consume logs from your Hubble export file path.
+Once you have configured the Hubble Exporter, you can configure your logging solution to consume logs from your Hubble export file path.
+
+To disable the static configuration, you must remove the ``hubble-export-file-path`` key in the ``cilium-config`` ConfigMap and manually clean up the log files created in the specified location in the container. The below command will restart the Cilium pods. If you edit the ConfigMap manually, you will need to restart the Cilium pods.
+
```
Comment (learnitall) [inline Documentation/observability/hubble-exporter.rst]: Here's an example of line wrap:

```suggestion
Once you have configured the Hubble Exporter, you can configure your logging solution to consume
logs from your Hubble export file path.

To disable the static configuration, you must remove the ``hubble-export-file-path`` key in
the ``cilium-config`` ConfigMap and manually clean up the log files created in the specified
location in the container. The below command will restart the Cilium pods. If you edit the
ConfigMap manually, you will need to restart the Cilium pods.
```

**Comment by ti-mo** [comment]
@saintdle Could you address the above feedback so we can get this merged? Thank you.

**Comment by saintdle** [comment]
> One small punctuation nit. Also, would you mind wrapping your lines? I'm not sure if we have a specific max line length, but it would be great if we could keep each line to a maximum of around 90 to 100 characters.
> 
> Otherwise LGTM! Thanks!

done, thanks for the help!

**Comment by youngnick** [comment]
@learnitall seems like this is ready for another review.

**Review by learnitall — APPROVED** [review]
Thanks!

**Comment by saintdle** [comment]
> Thanks!

Looks like it needed rebasing with main, which I did in Github UI, and now something else is wrong, can you help me fix?

**Comment by learnitall** [comment]
No worries @saintdle! I squashed your commits and rebased your changes onto `main`. Can you take a look and let me know if the commits look good to you? Thanks!

**Review by nathanjsweet — APPROVED** [review]

**Comment by ldelossa** [comment]
Docs only change with all required approvals. Merging.

**Pull request merged**